src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.tsx

```diff
@@ -96,7 +98,21 @@ const NewThreshold = ({ newThreshold }) => (
   </Block>
 )
 
-const SettingsDescription = ({ action, addedOwner, newThreshold, removedOwner }) => {
+const AddModule = ({ module }) => (
```

Comment:
If it's even possible to add a simple type here as unknown do it please, otherwise just ignore this comment

Reply:
yes, I can add it to all the other components in the file :man_shrugging: